neuralmonkey/tests/test_vocabulary.py

```diff
@@ -49,6 +49,7 @@ def test_there_and_back_self(self):
     def test_min_freq(self):
 
         vocabulary = Vocabulary()
+        vocabulary.correct_counts = True
```

Comment:
Klidně to revertni, zas tak žhavá featura to neni. Ale rozhodně by to nemělo být nastavovaný nikde jinde než v konstruktoru nebo těch továrničkovejch funkcích, tj. určitě ne v `add_tokenized_text`. Je to konstantní vlastnost. Existující slovík, co má `correct_counts == False`, tak se přece najednou z ničeho nic nemůže stát slovníkem s `correct_counts == True`.. To bys tam musel přidat správný počty ke všem existujícím slovům, co už v něm jsou.